df.units.xml

```diff
@@ -1803,7 +1815,21 @@
         </pointer>
         <int32_t name="unk_v4201_3" init-value='-1' since='v0.42.01'/>
         <int32_t name="unk_v4201_4" init-value='-1' since='v0.42.01'/>
-        <pointer is-array='true' type-name='int32_t' name='unk_v4410_1' comment='length is multiple of 9?'/>
+        <pointer name="memories">
+            <static-array type-name='unit_emotion_memory' count='8'/>
```

Comment:
Thoughts on giving these fields names? I've been trying to give most fields at least `unk_` names to avoid confusion with `anon_` names shifting around (I know that's a pain for large PRs, though...). Maybe `unk_slots_1` and `unk_slots_2` could work here.

Reply:
I thought about `shortterm`, `midterm`, `longterm`, or `shortterm`, `longterm` for the static-arrays and something special for the vector. I don't understand them well enough to give good names, so maybe obvious placeholders like you are suggesting are good enough and other people will be able to finish the research later.

I need to find matching names for the flags too. If placeholders names were used, would this be acceptable?

```diff
diff --git a/df.units.xml b/df.units.xml
index b3059aa..5175055 100644
--- a/df.units.xml
+++ b/df.units.xml
@@ -1750,8 +1750,9 @@
                     <flag-bit name='unk1'/>
                     <flag-bit name='unk2'/>
                     <flag-bit name='unk3'/>
-                    <flag-bit name='unk4'/>
-                    <flag-bit name='remembered' comment='Recollection of past experience rather than current'/>
+                    <flag-bit name='remembered2' comment='Recollection of past experience from memories.unk_slots_2 rather than current'/>
+                    <flag-bit name='remembered1' comment='Recollection of past experience from memories.unk_slots_1 rather than current'/>
+                    <flag-bit name='remembered3' comment='Recollection of past experience from memories.unk_slots_3 rather than current'/>
                 </bitfield>
                 <int32_t name='unk7'/>
                 <int32_t name='year'/>
```

Reply:
Done for most fields. Do I need to name the remaining `<int32_t/>` too?